nengo/tests/test_dists.py

```diff
@@ -134,6 +134,109 @@ def test_sqrt_beta(n, m):
     assert np.all(np.abs(np.asfarray(hist - expectation) / num_samples) < 0.16)
 
 
+@pytest.mark.parametrize("n,m", [(4, 1), (10, 5)])
+def test_sqrt_beta_analytical(n, m, seed):
+    """Tests pdf, cdf, and ppf of SqrtBeta distribution."""
+    pytest.importorskip('scipy')  # beta and betainc
+
+    dt = 0.001
+    x = np.arange(dt, 1+dt, dt)
+
+    dist = dists.SqrtBeta(n, m)
+
+    pdf = dist.pdf(x)
+    cdf = dist.cdf(x)
+    ppf = dist.ppf(cdf)
+
+    # The pdf should reflect the samples
+    np.random.seed(seed)
```

Comment:
You shouldn't have to do this. Make a `numpy.random.RandomState` and pass it where you need it.